geniza/footnotes/models.py

```diff
@@ -141,16 +140,49 @@ def __str__(self):
             else:
                 author = author_lastnames[0]
 
-        parts = []
-
         if self.title:
             # if this is an article, wrap title in quotes
             if self.source_type.type == "Article":
                 parts.append('"%s"' % self.title)
             else:
                 parts.append(self.title)
 
-        # TODO: formatted version with italics for book/journal title
+        if self.languages:
+            if not self.languages.all():
+                languages = None
+
+            if len(self.languages.all()) > 1:
+                # Separate languages with commas
+                languages = ",".join([l.name for l in self.languages.all()])
+            else:
+                languages = " ".join([l.name for l in self.languages.all()])
+
+        # Source formatting for given source type, see #252
```

Comment:
We don't typically reference issues in comments; I think because in the long term it's not a particularly helpful resource — any details that are needed to understand the code should be in the comments, which you've done.